demo/magic_8_ball/requirements.txt

```diff
@@ -0,0 +1,3 @@
+https://gradio-builds.s3.amazonaws.com/bed454c3d22cfacedc047eb3b0ba987b485ac3fd/gradio-4.40.0-py3-none-any.whl
```

Comment:
Remove hardcoded Gradio? Or do we need this for now?

Reply:
We need this until we publish the rc I guess